src/Water/vapor_pressure.jl

```diff
@@ -0,0 +1,156 @@
+"""
+    p_sat_water_ice(tem::FT)
+
```

Comment:
And here is where the common functions for saturation vapor pressure are defined
https://github.com/CliMA/ClimateMachine.jl/blob/766ad0c0354742b08baab5a9d8aef10a511d86c6/src/Common/Thermodynamics/relations.jl#L568

It would be great to use the same in the clouds and on land.

Reply:
It would indeed be good but make Land dependent on the Atmospheric model. Or should we pull Thermodynamics and (and SurfaceFluxes for that matter) out into their own entity (like the Paramaters?)

Reply:
@charleskawczynski was in the process of pulling Thermodynamics out?

Reply:
Yes, but I think it may be easier to iron out some of Thermo’s test profiles first. We should do this soon, though. Also, it may be easier to increase CLIMAParameter’s precision first, so that we can update the tests in one shot.